File: decompiled/sources/com/device/discovery/DeviceType.java
package com.device.discovery;

public enum DeviceType {
    DEVICETYPE_PC(0),
    DEVICETYPE_DVR(1),
    DEVICETYPE_IPC(2),
    DEVICETYPE_NVR(3);
    
    private int nCode;

    private DeviceType(int i) {
        this.nCode = i;
    }

    public String toString() {
        return String.valueOf(this.nCode);
    }
}

File: decompiled/sources/com/device/discovery/DeviceType.kt
package com.device.discovery

enum class DeviceType(private val nCode: Int) {
    DEVICETYPE_PC(0),
    DEVICETYPE_DVR(1),
    DEVICETYPE_IPC(2),
    DEVICETYPE_NVR(3);

    override fun toString(): String {
        return nCode.toString()
    }
}
